client/transaction-pool/src/metrics.rs

```diff
@@ -55,17 +54,10 @@ pub struct Metrics {
 impl Metrics {
 	pub fn register(registry: &Registry) -> Result<Self, PrometheusError> {
 		Ok(Self {
-			validations_scheduled: register(
+			submitted_transactions: register(
```

Comment:
Should we track rejected / removed transactions as well? That difference would give us current pool size (if tracked correctly).